src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -49,6 +73,56 @@ namespace ann /** Artificial Neural Network. */ {
  * \right.
  * @f}
  *
+ * For more information, read the following paper:
+ *
+ * @code
+ * @article{Clevert2015,
+ *   author  = {Djork{-}Arn{\'{e}} Clevert and Thomas Unterthiner and
+ *              Sepp Hochreiter},
+ *   title   = {Fast and Accurate Deep Network Learning by Exponential Linear
+ *              Units (ELUs)},
+ *   journal = {CoRR},
+ *   year    = {2015}
+ * }
+ * @endcode
+ *
+ *
+ * The SELU activation function is defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) &=& \left\{
+ *   \begin{array}{lr}
+ *    lambda * x & : x > 0 \\
+ *    lambda * alpha(e^x - 1) & : x \le 0
+ *   \end{array}
+ * \right. \\
+ * f'(x) &=& \left\{
+ *   \begin{array}{lr}
+ *     lambda & : x > 0 \\
+ *     lambda * (y + alpha) & : x \le 0
+ *   \end{array}
+ * \right.
+ * @f}
+ *
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @article{Klambauer2017,
+ *   author  = {Gunter Klambauer and Thomas Unterthiner and
+ *              Andreas Mayr},
+ *   title   = {Self-Normalizing Neural Networks},
+ *   journal = {Advances in Neural Information Processing Systems},
+ *   year    = {2017}
+ * }
+ * }
+ * @endcode
+ *
+ * NOTE:
```

Comment:
Can you use ``Note Make sure to use SELU activation function with normalized inputs and weights initialized with Lecun Normal Initialization.`` here, I think in this case doxygen is able to pick this up and highlight the text.